components/job-orchestration/job_orchestration/scheduler/query/query_scheduler.py

```diff
@@ -333,34 +448,51 @@ def get_archive_and_file_split_ids(
     return results
 
 
+@exception_default_value(default=[])
```

Comment:
```suggestion
@exception_default_value(default=False)
```

Default should be `False`, right?

Reply:
True.. Haven't looked at the code for a while and forgot what it is supposed to be